### PR TITLE
Exposed setters for sensors

### PR DESCRIPTION
**Pull request by DmitriySalnikov**
set_accelerometer, set_gravity, set_magnetometer and set_gyroscope exposed to use it in scripts.
For example I want to port [my module](https://github.com/DmitriySalnikov/GodotRemote) to GDNative but without this PR sensors data will not be available.

### CODE REVIEW

**Comment by DmitriySalnikov** [comment]
🐢

**Comment by akien-mga** [comment]
I'm not sure to understand what's the use case for having those methods exposed. Aren't those methods supposed to give information that comes from the OS itself? If I understand correctly, you'd want to have a proxy between a mobile device and your running instance on desktop with https://github.com/DmitriySalnikov/GodotRemote, so you need to pass the mobile OS sensor data to the Windows version? I guess that could make sense.

For the PR itself, there are a few changes needed:
- PRs should be made against the `master` branch in priority. This change would also be valid in `master` for 4.0 and future releases, so if it's only done in `3.2` you won't have the feature anymore in future Godot versions, which would be a regression.
- The commit log should be amended to be clearer and in plain English, see https://github.com/godotengine/godot/blob/master/CONTRIBUTING.md#format-your-commit-messages-with-readability-in-mind
- You should add documentation for the newly exposed methods (generate the template with `godot --doctool .`).

**Comment by naithar** [comment]
I'm not entirely sure I understand the use case as well as the need for this change in `Input` class.
This methods are also already exposed, aren't they? 

`master`
https://github.com/godotengine/godot/blob/2abe996414b8b551e69e29461de3ff1bcaf5a28f/core/input/input.h#L296-L300

`3.2`
https://github.com/godotengine/godot/blob/3dd57a22df1fe431b71f2f4229cbff1107744658/main/input_default.h#L255-L259

Casting `Input` to `InputDefault` wouldn't cover it?

**Comment by DmitriySalnikov** [comment]
@akien-mga  I got it. But I do it later.

@naithar these methods are only available in c++, but it cannot be used in GDScript or GDNative.
I want to use it for debugging in the editor via the GDNative library. It's currently only available via a regular module.

(right bottom screen is a smartphone with my module)
https://youtu.be/LbFcQnS3z3E?t=76
